2025-05-09 02:30:03,418 INFO    MainThread:3147107 [wandb_setup.py:_flush():67] Current SDK version is 0.19.9
2025-05-09 02:30:03,418 INFO    MainThread:3147107 [wandb_setup.py:_flush():67] Configure stats pid to 3147107
2025-05-09 02:30:03,418 INFO    MainThread:3147107 [wandb_setup.py:_flush():67] Loading settings from /sdf/home/c/carsmith/.config/wandb/settings
2025-05-09 02:30:03,418 INFO    MainThread:3147107 [wandb_setup.py:_flush():67] Loading settings from /sdf/home/c/carsmith/PILArNet-ML/3d_segment/wandb/settings
2025-05-09 02:30:03,418 INFO    MainThread:3147107 [wandb_setup.py:_flush():67] Loading settings from environment variables
2025-05-09 02:30:03,418 INFO    MainThread:3147107 [wandb_init.py:setup_run_log_directory():662] Logging user logs to /sdf/home/c/carsmith/PILArNet-ML/3d_segment/wandb/run-20250509_023003-my0ngmv9/logs/debug.log
2025-05-09 02:30:03,418 INFO    MainThread:3147107 [wandb_init.py:setup_run_log_directory():663] Logging internal logs to /sdf/home/c/carsmith/PILArNet-ML/3d_segment/wandb/run-20250509_023003-my0ngmv9/logs/debug-internal.log
2025-05-09 02:30:03,418 INFO    MainThread:3147107 [wandb_init.py:monkeypatch_ipython():581] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7f36c025af50>
2025-05-09 02:30:03,418 INFO    MainThread:3147107 [wandb_init.py:init():781] calling init triggers
2025-05-09 02:30:03,418 INFO    MainThread:3147107 [wandb_init.py:init():786] wandb.init called with sweep_config: {}
config: {'epochs': 5, 'batch_size': 16, 'lr': 0.001, 'temperature': 0.07, '_wandb': {}}
2025-05-09 02:30:03,418 INFO    MainThread:3147107 [wandb_init.py:init():809] starting backend
2025-05-09 02:30:03,418 INFO    MainThread:3147107 [wandb_init.py:init():813] sending inform_init request
2025-05-09 02:30:03,424 INFO    MainThread:3147107 [backend.py:_multiprocessing_setup():101] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2025-05-09 02:30:03,424 INFO    MainThread:3147107 [wandb_init.py:init():823] backend started and connected
2025-05-09 02:30:03,428 INFO    MainThread:3147107 [wandb_run.py:_label_probe_notebook():1267] probe notebook
2025-05-09 02:30:03,430 INFO    MainThread:3147107 [wandb_run.py:_label_probe_notebook():1277] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2025-05-09 02:30:03,430 INFO    MainThread:3147107 [wandb_init.py:init():915] updated telemetry
2025-05-09 02:30:03,445 INFO    MainThread:3147107 [wandb_init.py:init():939] communicating run to backend with 90.0 second timeout
2025-05-09 02:30:04,051 INFO    MainThread:3147107 [wandb_init.py:init():1014] starting run threads in backend
2025-05-09 02:30:04,357 INFO    MainThread:3147107 [wandb_run.py:_console_start():2454] atexit reg
2025-05-09 02:30:04,357 INFO    MainThread:3147107 [wandb_run.py:_redirect():2306] redirect: wrap_raw
2025-05-09 02:30:04,357 INFO    MainThread:3147107 [wandb_run.py:_redirect():2371] Wrapping output streams.
2025-05-09 02:30:04,357 INFO    MainThread:3147107 [wandb_run.py:_redirect():2394] Redirects installed.
2025-05-09 02:30:04,363 INFO    MainThread:3147107 [wandb_init.py:init():1056] run started, returning control to user process
2025-05-09 02:30:04,364 INFO    MainThread:3147107 [jupyter.py:save_ipynb():384] [no run ID] not saving jupyter notebook
2025-05-09 02:30:04,364 INFO    MainThread:3147107 [wandb_init.py:_pause_backend():546] [no run ID] pausing backend
2025-05-09 02:30:28,593 INFO    MainThread:3147107 [wandb_init.py:_resume_backend():551] [no run ID] resuming backend
2025-05-09 02:30:28,613 INFO    MainThread:3147107 [wandb_run.py:_finish():2189] finishing run carsmith-stanford-university/simclr_encoder_pretraining/my0ngmv9
2025-05-09 02:30:28,614 INFO    MainThread:3147107 [jupyter.py:save_history():460] not saving jupyter history
2025-05-09 02:30:28,614 INFO    MainThread:3147107 [jupyter.py:save_ipynb():384] not saving jupyter notebook
2025-05-09 02:30:28,614 INFO    MainThread:3147107 [wandb_init.py:_jupyter_teardown():563] cleaning up jupyter logic
2025-05-09 02:30:28,614 INFO    MainThread:3147107 [wandb_run.py:_atexit_cleanup():2419] got exitcode: 0
2025-05-09 02:30:28,614 INFO    MainThread:3147107 [wandb_run.py:_restore():2401] restore
2025-05-09 02:30:28,614 INFO    MainThread:3147107 [wandb_run.py:_restore():2407] restore done
2025-05-09 02:30:29,333 INFO    MainThread:3147107 [wandb_run.py:_footer_history_summary_info():4064] rendering history
2025-05-09 02:30:29,333 INFO    MainThread:3147107 [wandb_run.py:_footer_history_summary_info():4096] rendering summary
2025-05-09 02:30:29,333 INFO    MainThread:3147107 [wandb_run.py:_footer_sync_info():4025] logging synced files
